person_capture/updater.py

```
@@ -382,6 +384,28 @@ def stage_zip_update(repo: Path, branch: Optional[str] = None) -> Tuple[bool, st
     except Exception as e:
         return False, f"stage error: {e}", None
 
+def _discard_pending_zip_update(repo: Path, flag: Optional[Path] = None, staged: Optional[Path] = None) -> None:
+    """Remove a staged zip update without touching source files."""
+    try:
+        if flag is None:
+            flag = repo / "update_pending.json"
+        if flag.exists():
+            flag.unlink()
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)